### DISCUSSION TITLE
Python3 already in OSX vs Python3 installed by Brew

### DISCUSSION THREAD

**Discussion by duemme** [Homebrew/discussions#1329] [Getting started]
Hi,
I discovered that my OSX BigSure 11.2.3 has python3 installed. I think that it comes with Xcode (that Brew asked me to install).
The bin "python3" is in "/usr/bin/" and packages are in "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/site-packages/".
If I install python3 via Brew do I have conflicts with this python3 and pip3 already installed?
Thanks

**Comment by carlocab** [comment]
If you're on ARM, and use the Homebrew installer to install Homebrew, you have nothing to worry about.

If you're on Intel, as long as `pip3` is not in `/usr/local/bin` (it most likely isn't), you have nothing to worry about.

**Reply by duemme** [reply]
pip3 is not in /usr/local/bin, is in /usr/bin/.
Thanks

**Reply by duemme** [reply]
I installed python 3.9 with brew and pip3 from the new installation took the place of the other pip3. Maybe because zsh found it in /usr/local/bin that its the first folder in which it looks.
